Makefile

```diff
@@ -532,7 +545,7 @@ storage-provisioner-image: out/storage-provisioner-$(GOARCH) ## Build storage-pr
 .PHONY: kic-base-image
 kic-base-image: ## builds the base image used for kic.
 	docker rmi -f $(REGISTRY)/kicbase:$(KIC_VERSION)-snapshot || true
-	docker build -f ./hack/images/kicbase.Dockerfile -t $(REGISTRY)/kicbase:$(KIC_VERSION)-snapshot  --build-arg COMMIT_SHA=${VERSION}-$(COMMIT) --cache-from $(REGISTRY)/kicbase:$(KIC_VERSION) --target base ./hack/images
+	docker build -f ./hack/images/kicbase.Dockerfile -t $(KIC_BASE_IMAGE)-snapshot -t $(KIC_BASE_IMAGE) --build-arg COMMIT_SHA=${VERSION}-$(COMMIT) --cache-from $(KIC_BASE_IMAGE) --target base ./hack/images
```

Comment:
pull upstream to include the new changes here, 
https://github.com/kubernetes/minikube/pull/8819/files?file-filters%5B%5D=No+extension

Reply:
pulled + used variable instead of strings as part of the conflict resolution